### PR TITLE
record cpu/mem usage and podStatistics for child cluster

### PR DESCRIPTION
**Pull request by GeorgeGuo2018**
#### What type of PR is this?
record cpu/mem usage and podStatistics for mcls

#### What this PR does / why we need it:
to display the usage and load info of child cluster on dashboard

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by DanielXLee** [comment]
this pr enhances the status manager to collect more monitor metric

**Review by dixudx — CHANGES_REQUESTED** [review]
We already have `Allocatable` and `Capacity` in cluster status. I don't think we need to similar fields again for same purpose.

**Comment by GeorgeGuo2018** [comment]
> We already have `Allocatable` and `Capacity` in cluster status. I don't think we need to add similar fields for same purpose.
Allocatable and Usage have some differents:
Allocatable record how many resources can be used in the cluster ; 
while Usage record the actual resource usage of the cluster in real time.

File: pkg/features/features.go [hunk 1]
```diff
@@ -79,6 +79,13 @@ const (
 	// alpha: v0.9.0
 	// Setting on hub side.
 	FeedInventory featuregate.Feature = "FeedInventory"
+
+	// EnableMetric enable the collection of podMetric and nodeMetric from metric server
+	//
+	// owner: @dixudx
```
Comment (huxiaoliang) [inline pkg/features/features.go:85]: why owner is dixudx rather than you?

Comment (dixudx) [reply pkg/features/features.go:85]: Please use flag instead of feature gate.

Comment (GeorgeGuo2018) [reply pkg/features/features.go:85]: > Please use flag instead of feature gate.

ok, will change to flag.

Comment (GeorgeGuo2018) [reply pkg/features/features.go:85]: > why owner is dixudx rather than you?

ok, will change to myself

Comment (GeorgeGuo2018) [reply pkg/features/features.go]: > Please use flag instead of feature gate.

fixed

**Pull request closed without merging**